extensions_admin/pulp_rpm/extensions/admin/export.py

```diff
@@ -106,6 +109,7 @@ def __init__(self, context, renderer, name='run', description=DESC_GROUP_EXPORT_
         self.add_option(OPTION_START_DATE)
         self.add_option(OPTION_END_DATE)
         self.add_option(OPTION_EXPORT_DIR)
+        self.add_flag(FLAG_MANIFEST)
```

Comment:
Oh, it seems that a flag is different than an option? Is this a silly Okaara thing?